win32ss/user/ntuser/defwnd.c

```diff
@@ -487,7 +487,7 @@ DefWndGetIcon(PWND pWnd, WPARAM wParam, LPARAM lParam)
             hIconRet = UserGetProp(pWnd, gpsi->atomIconSmProp, TRUE);
             break;
         default:
-            break;
+            hIconRet = NULL;
```

Comment:
DEFAULT_UNREACHABLE seems preferable since this is dead code.

Reply:
Done. I didn't know that macro.